### PR TITLE
[CZID-9376] Support array inputs in manifests and bulk upload loaders

### PR DESCRIPTION
**Pull request by morsecodist**
Allows manifests to specify arrays of inputs with the same name and type. Necessary for bulk downloads.

Because I was blocked on github actions I also folded in required loaders for bulk downloads into this and tested everything end to end.

### CODE REVIEW

File: entities/scripts/seed/bulk-download.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    )
+
+    # redo above joining File on Entity by
+    file_ids: list[dict[str, str]] = []
+    for file in files:
+        if file.protocol == FileAccessProtocol.s3 and session.s3_local:
+            session.upsert_bucket(file.namespace)
+            print(f"Creating file {file.id} in {file.namespace}/{file.path}")
+            session.s3_local.put_object(Bucket=file.namespace, Key=file.path, Body="ABC")
+        if len(file_ids) < 4:
```
Comment (rzlim08) [inline entities/scripts/seed/bulk-download.py:37]: Just curious why this is limited on <4?

File: workflows/plugins/output_loaders/entities/consensus_genome_output.py [hunk 2]
```diff
@@ -44,17 +44,28 @@ async def load(
             taxon_id = res["taxa"][0]["id"]
             accession_id = res["accessions"][0]["id"]
         else:
-            taxon_id = entity_inputs["taxon"].entity_id
-            accession_id = entity_inputs["accession"].entity_id
+            taxon_input = entity_inputs["taxon"]
+            assert isinstance(taxon_input, EntityInput)
+            taxon_id = taxon_input.entity_id
```
Comment (rzlim08) [inline workflows/plugins/output_loaders/entities/consensus_genome_output.py:49]: would it be possible to name this taxon_entity_id or something? I think taxon_id might be confusing later. Same with accession_id.

File: workflows/scripts/seed/bulk-download.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    cg_workflow = session.create_or_fetch_entity(Workflow, name="bulk-download")
+    cg_workflow.default_version = version
+    cg_workflow.minimum_supported_version = version
+    session.add(cg_workflow)
+    session.commit()
+
+    workflow_uri = session.transfer_wdl("run.wdl", "bulk-download")
+    cg_workflow_version = session.create_or_fetch_entity(WorkflowVersion, version=version, workflow=cg_workflow)
+    cg_workflow_version.workflow_uri = workflow_uri
+    with TempCZIDWorkflowFile("manifest.yml", "bulk-download") as manifest_file:
```
Comment (rzlim08) [inline workflows/scripts/seed/bulk-download.py:25]: Is this in main branch already?

**Review by rzlim08 — APPROVED** [review]

File: entities/scripts/seed/bulk-download.py (hunk 1 above)
Comment (morsecodist) [reply entities/scripts/seed/bulk-download.py:37]: Just thought this made for a nice minimal input for local testing

File: workflows/scripts/seed/bulk-download.py (hunk 3 above)
Comment (morsecodist) [reply workflows/scripts/seed/bulk-download.py:25]: yes I tested before merging then merged in workflows and updated here

**Pull request merged**